express/blocks/download-cards/download-cards.css

```diff
@@ -0,0 +1,334 @@
+:root {
+  --dark-bg-color: #000B1D;
+  --header-height: 63px;
+}
+
+/* cards block styles - removing is ok, do not modify */
+main .section.cards-container {
+  background-color: #F1F3F4;
```

Comment:
There's a bit of a mix of color vars and hardcoded colors... could some of this be consolidated with what's in `styles.css`?